peekingduck/pipeline/nodes/model/yoloxv1/yolox_files/trt_model.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2022 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""TensorRT model for PeekingDuck"""
+
+from typing import Any, List, Tuple
+import numpy as np
+import tensorrt as trt  # pylint: disable=import-error
+import pycuda.driver as cuda  # pylint: disable=import-error
```

Comment:
Updated `weights_checksum.json` and added script to compute hash into `converters/` folder.
Added installation instructions section: the NVIDIA TensorRT Installation web page actually mentions the need to install PyCUDA. Since the instructions may change in future, so there is no need to add `pycuda` as an optional dependency for now.